internal/graph/schema/servicemaintenance.graphqls

```diff
@@ -30,6 +30,12 @@ type ValkeyMaintenanceUpdate implements ServiceMaintenanceUpdate {
 }
 
 type ValkeyMaintenance {
+	"Day of the week when the maintenance is scheduled."
+	dayOfWeek: Weekday
+
+	"Time of day when the maintenance is scheduled."
+	timeOfDay: TimeOfDay
+
```

Comment:
Since both of these MUST or MUST NOT have values, we could add them to a new type where they can be required, and have an optional field pointing to that type.

```
type MaintenanceWindow {
    dayOfWeek: Weekday!
    timeOfDay: TimeOfDay!
}
```

then have `window: MaintenanceWindow` in `ValkeyMaintenance` and `OpenSearchMaintenance`.

Reply:
Byttet til dette forslaget.